lib/modules/datasource/repology/index.ts

```
@@ -177,7 +177,7 @@ export class RepologyDatasource extends Datasource {
       throw err;
     }
 
-    logger.debug(
```

@rarkins I think we should throw the error up instead of catching here

---

To be caught by the datasource index function?

---

ok, throwing the Error. Sounds good to mw.

---

> To be caught by the datasource index function?

yes